static/monorail-2.0.yaml

```diff
@@ -4060,7 +4062,7 @@ paths:
   /workflows/tasks/{injectableName}:
     delete:
       description:
-        Delete the workflow task with the specified value of the injectableName property.
+        Delete the workflow task with a specific value of the injectableName property.
```

Comment:
Suggest to emphasize this is task **definition**, to avoid misunderstand it as task instance.